src/main/java/bhs/devilbotz/commands/auto/BalancePID.java

```diff
@@ -62,9 +64,13 @@ public void initialize() {
   @Override
   public void execute() {
     double error = balancePid.calculate(drive.getRoll(), 0);
-    double output = MathUtil.clamp(error, -0.5, 0.5);
+    double output = MathUtil.clamp(error, -0.25, 0.25);
```

Comment:
We slowed down the max speed when balancing to prevent overshoot

Reply:
I know you tested this, but just a general comment/reminder about setting drive train speeds.  Be careful about speed being too low, won't overcome friction, momentum.

Reply:
Hi Maciej, we tested this with the practice bot, but do need to change this for the competition bot.  I hope to tune it tonight at Salem.